From ff0ad2681b06e4e7e0eadc281d3efda171bd1f5e Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 31 Jul 2013 08:03:35 +0200
Subject: pc: Drop RHEL-6 USB device compat_prop full-path

Message-id: <1375201922-6794-6-git-send-email-armbru@redhat.com>
Patchwork-id: 52814
O-Subject: [RHEL-7 PATCH v3 05/15] pc: Drop RHEL-6 USB device compat_prop full-path
Bugzilla: 983991
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

Upstream added property full-path to USB devices in commit eeb0cf9
(v1.1) for migration compatibility.  RHEL-6 got the feature in commit
29b10d1 (6.2), but without the property, because 6.2 was the first
version capable of migrating USB devices.  Therefore, full-path needs
to remain on by default for all machine types.

Signed-off-by: Markus Armbruster <armbru@redhat.com>

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 5d57a76..aae891e 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -904,10 +904,6 @@ static QEMUMachine pc_machine_rhel700 = {
         .property = "vapic",\
         .value    = "off",\
     },{\
-        .driver   = TYPE_USB_DEVICE,\
-        .property = "full-path",\
-        .value    = "no",\
-    },{\
         .driver   = "virtio-blk-pci",\
         .property = "event_idx",\
         .value    = "off",\
